lib/transformer.js

```diff
@@ -177,11 +177,9 @@ class Transformer {
     for (const obj of interfaces.values()) {
       let source = obj.toString();
 
-      const implDir = path.relative(outputDir, obj.opts.implDir).replace(/\\/g, "/"); // fix windows file paths
-      let implFile = implDir + "/" + obj.name + this.ctx.implSuffix;
-      if (implFile[0] !== ".") {
-        implFile = "./" + implFile;
```

Comment:
Is this bit not necessary anymore?

Reply:
Thanks for catching it. It turns out path.relative() does not prepend the necessary "./" (contrary to my impression). I only tested with an impl path that's not directly under the output directory so didn't catch this edge case. Restored the "./" fix.